nilearn/plotting/img_plotting.py

```diff
@@ -30,12 +31,26 @@
                                                  cbar_outline_set_xy)
 from .._utils.ndimage import get_border_data
 from ..datasets import load_mni152_template
+from ..image import iter_img
 from .displays import get_slicer, get_projector
 from . import cm
 
 ################################################################################
 # Core, usage-agnostic functions
 
+def _alpha_cmap(color):
+    """ Return a colormap with the given color, and alpha going from
+        zero to 1.
+    """
+    red, green, blue = color[:3]
+    cmapspec = [(red, green, blue, 0.),
+                (red, green, blue, 1.), ]
+    cmap = LinearSegmentedColormap.from_list(
+        'alpha', cmapspec, plt.cm.LUTSIZE)
+    cmap._init()
+    cmap._lut[:, -1] = np.linspace(.7, .95, cmap._lut.shape[0])
+    cmap._lut[-1, -1] = 0
```

Comment:
You did not address this comment. What you can do is directly ask yannick or even do a `git blame` where you found the code and look at the log of the corresponding commit. It may be helpful.